include/cutlass/gemm/kernel/xe_gemm.hpp

```diff
@@ -162,11 +162,28 @@ class GemmUniversal<
     auto m = get<0>(args.problem_shape);
     auto n = get<1>(args.problem_shape);
     auto k = get<2>(args.problem_shape);
+    auto l = get<3>(args.problem_shape);
+    bool is_batch = l > 1;
+    auto check_stride = [is_batch](auto stride, int el_size){
+      auto a = get<0>(stride);
+      auto b = get<1>(stride);
+      auto valid_is_unit = a == _1{} || b == _1{};
+      auto inner = a == _1{} ? b : a;
+      auto valid_inner = inner % (16 / el_size) == 0;
```

Comment:
```suggestion
      auto valid_inner = inner > 0 && inner % (16 / el_size) == 0;
```
I ran into this today - if the user forgets to initialize the dynamic part of the stride, I think it'll be zero, which would pass the mod test.

Reply:
I would say 0 stride is a weird, bit technically correct case.